### ISSUE TITLE
Resolution change resets brightness to 100%

### ISSUE THREAD

**Issue by Anatharias** [xanderfrangos/twinkle-tray#431]
Hello,
I use [HRC](https://funk.eu/hrc/) to switch resolution while I remote to my computer.

I also use the following Powershell command to invoke a DPI change.
```# $scaling = 0 : 100% (default)
# $scaling = 1 : 125% 
# $scaling = 2 : 150% 
# $scaling = 3 : 175% 
# $scaling = 4 : 200% 
param($scaling = 0)
$source = @’
[DllImport("user32.dll", EntryPoint = "SystemParametersInfo")]
public static extern bool SystemParametersInfo(
                 uint uiAction,
                 uint uiParam,
                 uint pvParam,
                 uint fWinIni);
‘@
$apicall = Add-Type -MemberDefinition $source -Name WinAPICall -Namespace SystemParamInfo –PassThru
$apicall::SystemParametersInfo(0x009F, $scaling, $null, 1) | Out-Null```

And each time I change resolution, brightness is pushed to maximum.
I use latest v1.14.5
Thanks

**Comment by xanderfrangos** [comment]
So, this is expected behavior (sort of). After a resolution or refresh rate change, the brightness usually resets to the last manually-set brightness (meaning it was set with the hardware controls, not Twinkle Tray). I think it happens to all monitors. However, Twinkle Tray is supposed to apply the last known brightness 0.75s and 5s after a detected change with the displays (hardware connection change or res/refresh change). 

Is the brightness not being re-applied after a resolution change?

**Comment by Anatharias** [comment]
Yes, this is it. Brightness on the monitor is set at 35% and contrast 75% (AW3821DW), and once I change resolution, in Twinkle-Tray it goes to 100% brightness, even though I checked the checkbox to retain the brightness. it does that with one or two monitors attached.
Note that I never set the brightness to 100%. I use HDR from times to times, and this behaviour is the same with HDR enabled under windows.
